auth_backends/tests/test_pipeline.py

```diff
@@ -78,7 +78,9 @@ def test_update_email_with_none(self, mock_set_attribute, mock_toggle):
 
         mock_set_attribute.assert_any_call('update_email.username_mismatch', False)
         mock_set_attribute.assert_any_call('update_email.rollout_toggle_enabled', False)
-        # email_updated should not be called since email wasn't updated
+        # Verify email_updated was not set
+        for call_args in mock_set_attribute.call_args_list:
+            self.assertNotEqual(call_args[0][0], 'update_email.email_updated')
```

Comment:
It's unclear if `call_args[0][0]` is correct or not. As noted in the other column, if you use the same method to assert elsewhere that this call is found, it would make it easier to trust that it not being found is not a false positive.

Reply:
This can be marked resolved.